common.go

```diff
@@ -96,7 +96,6 @@ const (
 	extensionSignatureAlgorithmsCert uint16 = 50
 	extensionKeyShare                uint16 = 51
 	extensionNextProtoNeg            uint16 = 13172 // not IANA assigned
-	extensionALPS                    uint16 = 17513
```

Comment:
This library generally does not modify the standard lib files.  I removed this in favor of the `fakeExtensionALPS` value I added to `u_common.go`.

Reply:
Looks good. Thanks!